install/variables.tf

```diff
@@ -105,7 +105,7 @@ variable "openshift_pull_secret" {
 variable "openshift_installer_url" {
   type        = string
   description = "The URL to download OpenShift installer."
-  default     = "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest"
+  default     = "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/"
 }
```

Comment:
This URL is returning a redirect from https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest to https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ (same URL with an extra `/` character at the end) and `wget` cannot follow it.